client/src/components/Meal/MealBox/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./style.css";
+<<<<<<< HEAD
```

Comment:
Looks like there was a merge conflict that started here that didn't get resolved?